content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,339 @@
+
+Istio has two webhooks (Galley and Sidecar Injector). In current Istio implementation,
+Galley and Sidecar Injector manage their own webhook configurations.
+This task uses `istioctl`, instead of Galley and Sidecar Injector, to manage the webhook configurations
+of Galley and Sidecar Injector.
+
+## Install Istio
+
+1.  The yaml file [`values-istio-dns-cert.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml)
+    contains an example configuration for provisioning the DNS certificates used in webhook configurations.
+    Install Istio with [DNS certificate configured](/docs/tasks/security/dns-cert) using [Helm](/docs/setup/install/helm/#prerequisites):
+
+    {{< text bash >}}
+    $ kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value core/account)"
+    $ kubectl create namespace istio-system
+    $ helm template install/kubernetes/helm/istio-init --name istio-init --namespace istio-system | kubectl apply -f -
+    $ helm template \
+        --name=istio \
+        --namespace=istio-system \
+        --set global.operatorManageWebhooks=true \
+        --values install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml \
+        install/kubernetes/helm/istio > istio-webhook-management.yaml
+    $ kubectl apply -f istio-webhook-management.yaml
+    {{< /text >}}
+
+## Check webhook certificates
+
+The webhook certificates generated are stored in the secrets specified in the configuration.
+
+1.  Check that webhook certificates have been generated:
+
+    {{< text bash >}}
+    $ kubectl get secret dns.istio-galley-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+    $ kubectl get secret dns.istio-sidecar-injector-service-account -n istio-system -o json | jq -r '.data["cert-chain.pem"]' | base64 --decode | openssl x509 -in - -text -noout
+    {{< /text >}}
+
+    The output from the above commands should include the DNS names of Galley and Sidecar Injector, respectively:
+
+    {{< text plain >}}
+    X509v3 Subject Alternative Name:
+      DNS:istio-galley.istio-system.svc, DNS:istio-galley.istio-system
+    {{< /text >}}
+
+    {{< text plain >}}
+    X509v3 Subject Alternative Name:
+      DNS:istio-sidecar-injector.istio-system.svc, DNS:istio-sidecar-injector.istio-system
+    {{< /text >}}
+
+## `istioctl` enables webhook configurations
+
+1.  Generate `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration` by running the following
+command:
+
+    {{< text bash >}}
+    $ helm template \
+        --name=istio \
+        --namespace=istio-system \
+        --values install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml \
+        install/kubernetes/helm/istio > istio-webhook-config.yaml
+    {{< /text >}}
+
+1.  From `istio-webhook-config.yaml`, search `'kind: MutatingWebhookConfiguration'` and save
```

Comment:
If this is for more then experimental we need to improve the UX here. If its is just experimental we should make that more clear

Reply:
Added TODOs to improve the UX here. The document has a warning at the top saying this is an experimental feature.